apps/vendors/views.py

```
@@ -0,0 +1,91 @@
+    ]
+
+    def get_queryset(self):
+        user = self.request.user
+        queryset = Vendor.objects.select_related('user')
+        if user.is_staff or user.is_superuser:
+            return queryset
+        if hasattr(user, 'vendor_profile'):
+            return queryset.filter(pk=user.vendor_profile_id)
+        return queryset.filter(is_active=True, is_verified=True)
+
+    def _assert_can_mutate(self, vendor):
```

Ruff (0.14.0)

Missing return type annotation for private function `_assert_can_mutate`

Add return type annotation: `None`

(ANN202)